README.rst

```diff
@@ -1,23 +1,22 @@
-SQLAlchemy Sphinx 
+SQLAlchemy Sphinx
 =================
 
-|travis| |pypi| |coveralls|
+|travis| |coveralls|
 
-.. |travis| image:: https://travis-ci.org/AdrielVelazquez/sqlalchemy-sphinx.svg?branch=master
-    :target: https://travis-ci.org/AdrielVelazquez/sqlalchemy-sphinx
-    
-.. |pypi| image:: https://badge.fury.io/py/sqlalchemy-sphinx.svg
-    :target: https://badge.fury.io/py/sqlalchemy-sphinx
-    
-.. |coveralls| image:: https://coveralls.io/repos/github/AdrielVelazquez/sqlalchemy-sphinx/badge.svg?branch=master 
-    :target: https://coveralls.io/github/AdrielVelazquez/sqlalchemy-sphinx?branch=master
+.. |travis| image:: https://travis-ci.org/akissa/sqlalchemy-sphinx.svg?branch=py26
```

Comment:
Switched travis and coveralls to a personal url, do you mind converting that back.